src/LocalField/Conjugates.jl

```diff
@@ -53,15 +51,17 @@ function newton_lift(f::ZZPolyRingElem, r::LocalFieldElem, precision::Int = pare
   for p = reverse(chain)
     r = setprecision!(r, p)
     o = setprecision!(o, p)
-    setprecision!(Q, p)
     setprecision!(qf, p)
     setprecision!(qfs, p)
-    r = r - qf(r)*o
-    if Nemo.precision(r) >= n
-      setprecision!(Q, n)
+    r = with_precision(Q, p) do
+      return r - qf(r)*o
+    end
+    if precision(r) >= n
```

Comment:
It seems to me that before, the precision of `r` resp. `Q` was always restricted to `n` if `precision(r) >= n`, while now it might stay at a larger value. Presumably this is fine, though, or even an improvement?

Unfortunately this particular case does not seem to be covered by CI.